lib/SodiumCrypto.js

```diff
@@ -54,6 +59,18 @@ export default class SodiumCrypto extends Crypto {
     return sodium.crypto_box_open_easy(eak, nonce, publicKey, privateKey)
   }
 
+  async decryptNoteEak(readerKey, encryptedAk, writerKey) {
```

Comment:
EAKs are the exact same format as records -- I may have missed the nuance, but what is the advantage of a note specific decrypt method on the Crypto object?

Reply:
DecryptEAK decrypts an EAKInfo object which can be artificially created; but it is only created as a response from the server which injects additional un-needed fields for this operation AFAIK.
```
export default class EAKInfo extends Serializable {
  constructor(eak, authorizerID, authorizerPublicKey, signerId, signerSigningKey) {
    super()

    this.eak = eak
    this.authorizerID = authorizerID
    this.authorizerPublicKey = new PublicKey(authorizerPublicKey)
    this.signerId = signerId
    this.signerSigningKey = new SigningKey(signerSigningKey)
  }

```